tests/test/lib/test_sgd.cpp

```diff
@@ -2,312 +2,334 @@
 
```

Comment:
Reformatted this as part of my clang-format experimentation. I'll do a single PR to run it across the whole codebase soon.

Reply:
I know we are still waiting for some extra work to be merged, but I'd rather do this in a once-and-for-all manner. Or at least group style-only changes in style-only PRs. This way we can be sure nothing is slipping through the net...

Reply:
:+1: